chart/templates/iq-server-deployment.yaml

```diff
@@ -191,6 +191,21 @@ spec:
         - name: {{ .Release.Name }}-fluentd-container
           image: {{ .Values.fluentd.image.repository }}:{{ .Values.fluentd.image.tag }}
           imagePullPolicy: {{ .Values.fluentd.image.pullPolicy }}
+          resources:
+            requests:
+              {{- if .Values.fluentd.resources.requests.cpu }}
+              cpu: {{ .Values.fluentd.resources.requests.cpu }}
+              {{- end }}
+              {{- if .Values.fluentd.resources.requests.memory }}
+              memory: {{ .Values.fluentd.resources.requests.memory }}
+              {{- end }}
+            limits:
+              {{- if .Values.fluentd.resources.limits.cpu }}
+              cpu: {{ .Values.fluentd.resources.limits.cpu }}
+              {{- end }}
+              {{- if .Values.fluentd.resources.limits.memory }}
+              memory: {{ .Values.fluentd.resources.limits.memory }}
+              {{- end }}
```

Comment:
should we be including this whole section only based on `{{- if .Values.hpa.enabled }}` ?  With that we could even consider defining some reasonable defaults to these limits